services/autoscaling/tests/unit/test_cli.py

```diff
@@ -0,0 +1,21 @@
+# pylint:disable=unused-variable
+# pylint:disable=unused-argument
+# pylint:disable=redefined-outer-name
+
+
+from simcore_service_autoscaling.cli import main
+from typer.testing import CliRunner
+
+runner = CliRunner()
+
+
+def test_settings():
+    result = runner.invoke(main, ["settings"])
+    assert result.exit_code == 0
+    assert "APP_NAME=simcore-service-autoscaling" in result.stdout
```

Comment:
do we not have a standardised test for this @pcrespov ?

Reply:
so if you have please point me there. This test I took directly from the typer website.

Reply:
I'll let @pcrespov answer this, since I might be confusing with something else.